skimage/segmentation/setup.py

```diff
@@ -23,6 +24,9 @@ def configuration(parent_package='', top_path=None):
                          include_dirs=[get_numpy_include_dirs()])
     config.add_extension('_slic', sources=['_slic.c'],
                          include_dirs=[get_numpy_include_dirs()])
+    config.add_extension('_remap', sources='_remap.cpp',
+                         include_dirs=[get_numpy_include_dirs()],
+                         language='c++', extra_compile_args=['-std=c++0x'])
```

Comment:
Just btw, this arg is required because `std::unordered_map` is not enabled by default in some older versions of gcc, and we were hitting this is the Linux builds.

https://stackoverflow.com/a/3973692/224254

I'll add a comment.